include/fix8/sessionwrapper.hpp

```diff
@@ -248,10 +248,16 @@ class ClientSession : public ClientSessionBase
 	/// Dtor.
 	virtual ~ClientSession ()
 	{
-		delete _persist;
+		session_ptr()->clear_connection( _cc );
 		delete _session;
-		delete _log;
+		_session = nullptr;
+		_persist->stop();
```

Comment:
The persister shouldn't need to be explicitly stopped here in the dtor. Explain why you think it is needed?